src/applications/model/gaming-streaming-server.h

```diff
@@ -92,34 +88,41 @@ class GamingStreamingServer : public Application
   void EraseStatistics (void);
 
 protected:
+  /**
+   * \brief Add parameters of new stream
+   *
+   * \param packetSize A random number generator for packet size
+   * \param interArrivalTime A random number generator for inter-arrival time
+   */
+  void AddNewTraffic (Ptr<RandomVariableStream> packetSize, Ptr<RandomVariableStream> interArrivalTime);
```

Comment:
no need to make it protected, you could still allow users to create custom traffic patterns (e.g., in `scratch/evaluate_gaming_traffic.cc`)